nemo/collections/tts/parts/transformer/sublayers.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import torch.nn.functional as F
+from torch import nn
+
+from nemo.collections.tts.parts.transformer import modules
+
+
+class MultiHeadAttention(nn.Module):
```

Comment:
Ok I know for sure, this class already exists. Please use the nlp module

Reply:
Well, input, internals and output are slightly different for other class. I am not sure we can merge them w/o problems.

Reply:
Please find a way to merge them. I do not see the point in creating multiple transformer modules.

Reply:
Done.